WalletWasabi/Services/UpdateManager.cs

```diff
@@ -17,7 +19,8 @@ namespace WalletWasabi.Services;
 public class UpdateManager : IDisposable
 {
 	private string InstallerPath { get; set; } = "";
-	private const byte MaxTries = 3;
+	private string Sha256SumsFilePath { get; set; } = "";
```

Comment:
This should not be here. It seems like there is a flaw in the design somewhere which is causing you to need to store the value, not in a local variable.

Reply:
It's used in two places as well, this makes it more accessable and bypass the problem that we have to pass it down from a root function to two different paths, more deeper. But I changed it in a0525c673d52823280798bcd8ca5270376f1e496 , does it suit the need?